cmd/kube-scheduler/app/server.go

```diff
@@ -58,6 +58,8 @@ import (
 	"k8s.io/klog"
 )
 
+var schedulerRegistry = framework.NewRegistry()
```

Comment:
Just a note, if there is more than one registry, all registry use ``schedulerRegistry`` variable, it may cause race condition that somewhere uses ``schedulerRegistry`` in a goroutine.

Reply:
Yes, if so, we need add a lock for function `Register` at https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/v1alpha1/registry.go#L35. I could add it later.